lib/linguist/generated.rb

```diff
@@ -274,6 +275,18 @@ def generated_protocol_buffer?
       return lines[0].include?("Generated by the protocol buffer compiler.  DO NOT EDIT!")
     end
 
+    # Internal: Is the blob a Javascript source file generated by the
+    # Protocol Buffer compiler?
+    #
+    # Returns true of false.
+    def generated_javscript_protocol_buffer?
+      return false unless extname == ".js"
+      return false unless lines.count > 6
+
+      return lines[5].include?("GENERATED CODE -- DO NOT EDIT!")
```

Comment:
Is this text always on the fifth line? Couldn't there be more (or less) annotations such as `@fileoverview` and `@public`, or even no comment lines before?